packages/core/util/io/index.js

```diff
@@ -1,5 +1,7 @@
 import { LocalFile, BlobFile } from 'generic-filehandle'
-import { openUrl } from './rangeFetcher'
+import { openUrl as rangeFetcherOpenUrl } from './rangeFetcher'
+
+export const openUrl = rangeFetcherOpenUrl
```

Comment:
Just curious what the re-renaming is for here

Reply:
that's so it can re-export the openUrl from rangeFetcher.  can't just `export openUrl` for some reason

Reply:
I think you can do `export { openUrl }`